firestore/README.md

```diff
@@ -227,7 +243,155 @@ FirestoreRecyclerAdapter adapter = new FirestoreRecyclerAdapter<Chat, ChatHolder
 ```
 
 
+### Using the `FirestorePagingAdapter`
+
+The `FirestorePagingAdapter` binds a `Query` to a `RecyclerView` by loading documents in pages.
+This results in a time and memory efficient binding, however it gives up the real-time events
+afforted by the `FirestoreRecyclerAdaoter`.
+
+The `FirestorePagingAdapter` is built on top of the [Android Paging Support Library][paging-support].
+Before using the adapter in your application, you must add a dependency on the support library:
+
+```groovy
+implementation 'android.arch.paging:runtime:1.0.0-beta1'
```

Comment:
Should we use `1.x.x` instead of a specific version since we're going to forget to update it?